index.js

```diff
@@ -62,9 +56,16 @@ GitHub.prototype = {
       cb = data;
       data = null;
     }
+
+    cb = typeof cb === 'function' ? cb : function noop () {};
```

Comment:
Know that, but it wouldn't be possible to check if it's okey.

Also, istanbul is mostly used coverage tool and it not detects that things, so that's why.